dev-lisp/newlisp/newlisp-10.7.5.recipe

```diff
@@ -0,0 +1,55 @@
+SUMMARY="Put the fun back in Lisp"
+DESCRIPTION="newLISP is a LISP-like scripting language for doing things"
+HOMEPAGE="https://www.newlisp.org"
+COPYRIGHT="2013 Lutz Mueller"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="http://www.newlisp.org/downloads/newlisp-10.7.5.tgz"
+SOURCE_FILENAME="newlisp-$portVersion.tar.gz"
+CHECKSUM_SHA256="dc2d0ff651c2b275bc4af3af8ba59851a6fb6e1eaddc20ae75fb60b1e90126ec"
+PATCHES="newlisp-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="$portVersion"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+       newlisp$secondaryArchSuffix = $portVersion
+       cmd:newlisp = $portVersion
+       cmd:newlisp_$portVersion = $portVersion
+       cmd:newlispdoc = $portVersion
+       "
+
+
+REQUIRES="
+       haiku$secondaryArchSuffix
+       lib:libncursesw$secondaryArchSuffix
+       lib:libreadline$secondaryArchSuffix
+       "
+
+BUILD_REQUIRES="
+       haiku${secondaryArchSuffix}_devel
+       devel:libncursesw$secondaryArchSuffix
+       devel:libreadline$secondaryArchSuffix
+       "
+
+BUILD_PREREQUIRES="
+       cmd:gcc$secondaryArchPrefix
+       cmd:make
+       cmd:pkg_config$secondaryArchPrefix
```

Comment:
Was getting parse errors with this (I only see them when gcc2 is involved, so thought something was wrong here :)
```diff
-       cmd:gcc$secondaryArchPrefix
+	cmd:gcc$secondaryArchSuffix
       cmd:make
-       cmd:pkg_config$secondaryArchPrefix
+	cmd:pkg_config$secondaryArchSuffix
```